docs/asciidoc/graph-updates/ttl.adoc

```diff
@@ -79,15 +79,15 @@ image::play-movies.png[title="Movies Graph Visualization"]
 [[ttl-expireAt]]
 === Expire node(s) at specified time
 
-The `apoc.ttl.expireAtInstant` procedure deletes a node or group of nodes after the datetime specified.
+The `apoc.ttl.expire` procedure deletes a node or group of nodes after the datetime specified.
```

Comment:
This is a breaking change, I think we need to guarantee the backwards compatibility.

Reply:
I am not sure I get it, there is no procedure expireAtInstant.

Reply:
Oh sorry I missed that, thank you!

Reply:
This has been fixed already in 4.1. Removing from this pr.